docs/building_and_running.md

```diff
@@ -41,6 +41,13 @@ Open `build\BigWheels.sln` and build
 
 Built binaries are written to `build\bin`.
 
+## Android (on Windows or Linux)
+```
+git clone --recursive https://github.com/google/BigWheels
+```
+
+Use Android Studio to open `android` subfolder and build
```

Comment:
do we still need to use VS to build the samples as lib? or the Android Studio would build the lib files as well then pack it into apk?

Reply:
No. We do not need VS at all. Android Studio does all the build stuff for you. You don't even need to use cmake directly.